content/techniques/task-scheduling.md

```diff
@@ -53,13 +53,32 @@ export class TasksService {
 
 In this example, the `handleCron()` method will be called each time the current second is `45`. In other words, the method will be run once per minute, at the 45 second mark.
 
+Some of you might consider defining a simple cron pattern with a plain string is a cumbersome task to be done. You can use the `CronExpression` enum instead, which provides commonly used cron patterns, as follows:
```

Comment:
I have a few suggestions.

1. For this paragraph, I suggest the following rewording

From: "Some of you might consider defining a simple cron pattern with a plain string is a cumbersome task to be done. You can use the `CronExpression` enum instead, which provides commonly used cron patterns, as follows:"

To: "The @nestjs/schedule package provides a convenience enum with commonly used cron patterns.  You can use this enum as follows:"

2. I think the section flows better if we move this documentation to the bottom of the section, just  after the section with the samples (Starting with "Some sample cron patterns are" and ending with the last sample).

**Question**: Is there an external reference where these are documented?  (I.e., is this a well known set of patterns, or is this something you created from scratch?).  The reason I'm asking is that if there is a good external reference, we should cite it here to help people know the available enum values.  It is probably too much to add these directly into the doc, so I'd rather do a link if one exists.  If not, that's fine too. :)

Reply:
@johnbiundo Although I've thought carefully about the section flows before deciding where to put it, the idea wasn't good enough. 😆 (Also, I like your phrase! I'll revise it.)

In response to your question, finding references for commonly used cron expressions was difficult. So I've written down the patterns that came to my mind one by one, which considered being simple and frequently being used.

Just an FYI, I didn't write complex patterns on purpose.